src/runtime-config/runtime-config.controller.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Body,
+  Controller,
+  Get,
+  Param,
+  Put,
+  Req,
+  UseGuards,
+} from "@nestjs/common";
+import { ApiBearerAuth, ApiBody, ApiTags } from "@nestjs/swagger";
+import { Request } from "express";
+import { AllowAny } from "src/auth/decorators/allow-any.decorator";
+import { JWTUser } from "src/auth/interfaces/jwt-user.interface";
+import { OutputRuntimeConfigDto } from "./dto/runtime-config.dto";
+import { RuntimeConfigService } from "./runtime-config.service";
+import { PoliciesGuard } from "src/casl/guards/policies.guard";
+import { Action } from "src/casl/action.enum";
+import { AppAbility } from "src/casl/casl-ability.factory";
+import { CheckPolicies } from "src/casl/decorators/check-policies.decorator";
+import { RuntimeConfig } from "./schemas/runtime-config.schema";
+@ApiBearerAuth()
+@ApiTags("runtime-config")
+@Controller("runtime-config")
+export class RuntimeConfigController {
+  constructor(private readonly runtimeConfigService: RuntimeConfigService) {}
+
+  @AllowAny()
+  @Get("data/:id")
```

Comment:
`data` strikes me as a surprising choice for the endpoint path. I see a close link between this functionality and the /admin/config endpoint. Maybe it could just be `GET/PUT/PATCH /admin/config/:id`? And there should be a way to get either the merged configuration or just the overridden values.